R/datasets.R

```diff
@@ -1,58 +1,101 @@
-#' Coordinates of Pennsylvania survey sites
+#' Data: sites x locations object
```

Comment:
Maybe add the dataset detail in the title like:
`Data: sites x locations of Western Europe mammals occurrence`

Reply:
Ok, I will do it (as for other datasets).